dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -1429,6 +1443,17 @@ private void wesCancelHelp() {
         printHelpFooter();
     }
 
+    private void wesServiceInfoHelp() {
+        printHelpHeader();
+        out("Usage: dockstore " + getEntryType().toLowerCase() + " wes service-info --help");
```

Comment:
The entry type is irrelevant for this endpoint.

But it opens a can of worms because `wes` always follows the entry type, so leave it for now.